third_party/2and3/pymysql/cursors.pyi

```diff
@@ -1,8 +1,9 @@
-from typing import Any, Dict, Iterator, List, Optional, Text, Tuple, TypeVar, Union
+from typing import Any, Dict, Iterator, List, Optional, Text, Tuple, TypeVar, Union, Iterable
 
 from .connections import Connection
 
-Gen = Union[Tuple[Any, ...], Dict[str, Any]]
+Gen = Union[Tuple[Any, ...], Dict[Text, Any]]
+QueryArgs = Union[Tuple[Any, ...], List[Any], Dict[Text, Any], None]
```

Comment:
As `QueryArgs` is a stubs-only definition, it should be named `_QueryArgs`. Same for `Gen`, but as that predates your PR, you can leave it alone if you want to.

Reply:
Prefixed both with an underscore.